src/plugins/GUITestBase/src/GTUtilsOptionPanelMSA.cpp

```diff
@@ -86,11 +86,13 @@ void GTUtilsOptionPanelMsa::toggleTab(HI::GUITestOpStatus &os, GTUtilsOptionPane
 #undef GT_METHOD_NAME
 
 #define GT_METHOD_NAME "openTab"
-void GTUtilsOptionPanelMsa::openTab(HI::GUITestOpStatus &os, Tabs tab) {
+QWidget *GTUtilsOptionPanelMsa::openTab(HI::GUITestOpStatus &os, Tabs tab) {
```

Comment:
Wow, that's strange. Rename it as `getOpenedTab` or something like this

Reply:
Wow, it is openTab, not just a "get"